grakn-graql/src/test/java/ai/grakn/graql/internal/query/InsertQueryTest.java

```diff
@@ -381,7 +382,11 @@ public void testErrorWhenInsertWithMultipleIds() {
     @Test
     public void testErrorWhenInsertWithMultipleValues() {
```

Comment:
I know it's not related to this PR but you have a bunch of crappy test names which could use your love and attention.